docs/Create-Packages/Dependency-Versions.md

```diff
@@ -99,14 +99,6 @@ A few examples:
   <dependency id="ExamplePackage" version="[1.3.2,1.5)" />
 ```
 
-
-If no version is specified for a dependency, NuGet behaves as follows:
-
-- NuGet v2.7.2 and earlier: The **latest** package version will be used
-- NuGet v2.8 and later:  The **lowest** package version will be used
```

Comment:
I think 2.8.x also takes the highest version of the dependency. It was in 3.0.0 that this changed from what I remember. Have you tried this to verify?